gui/wxpython/location_wizard/wizard.py

```diff
@@ -1611,78 +1646,52 @@ def OnPageChanging(self, event):
                     self.parent.datum_trans = dtrans
             self.GetNext().SetPrev(self)
 
-    def OnText(self, event):
-        self.epsgcode = event.GetString()
-        try:
-            self.epsgcode = int(self.epsgcode)
-        except:
-            self.epsgcode = None
-
+    def EnableNext(self, enable=True):
         nextButton = wx.FindWindowById(wx.ID_FORWARD)
+        nextButton.Enable(enable)
 
-        if self.epsgcode and self.epsgCodeDict and \
-                self.epsgcode in self.epsgCodeDict.keys():
-            self.epsgdesc = self.epsgCodeDict[self.epsgcode][0]
-            self.epsgparams = self.epsgCodeDict[self.epsgcode][1]
-            if not nextButton.IsEnabled():
-                nextButton.Enable(True)
-        else:
-            self.epsgcode = None  # not found
-            if nextButton.IsEnabled():
-                nextButton.Enable(False)
-            self.epsgdesc = self.epsgparams = ''
-
-    def OnSearch(self, event):
+    def OnTextChange(self, event):
         value = self.searchb.GetValue()
-
         if value == '':
+            self.tlink.SetURL(str("https://epsg.io/"))
             self.epsgcode = None
             self.epsgdesc = self.epsgparams = ''
-            self.tcode.SetValue('')
-            self.searchb.SetValue('')
             self.OnBrowseCodes(None)
+            self.EnableNext(False)
         else:
-            try:
-                self.epsgcode, self.epsgdesc, self.epsgparams = \
-                    self.epsglist.Search(index=[0, 1, 2], pattern=value)
-            except (IndexError, ValueError):  # -> no item found
+            self.tlink.SetURL(str("https://epsg.io/?q={0}".format(value)))
```

Comment:
What about ```https://epsg.io/{0}``` directly ?

Reply:
Hm, this will work for EPSG (like "5514") but for a keyword (like "krovak"). Marking this comment as not relevant.